### ISSUE TITLE
Syntax error in boot module

### ISSUE THREAD

**Issue by mpmanti** [joshuafontany/TW5-JsonMangler#13]
> Well, this is embarrassing. It is recommended that you restart TiddlyWiki by refreshing your browser
> Syntax error in boot module $:/core/modules/wiki.js:10993: eval@[native code]

Here are the steps that I took:

1. Download empty.html from TiddlyWiki.com via Safari for iOS. TW version is v5.1.22.
2. Copy empty.html to wikis folder in Quine 2.
3. Open empty.html in Quine 2.
4. Install ModLoader via drag-and-drop.
5. Save wiki.
6. Refresh wiki.
7. Install JsonMangler via drag-and-drop.
8. Save wiki.
9. Refresh wiki.
10. Close wiki.
11. Open wiki.
12. See error message every time I reopen the wiki.

I haven't tested it on platforms other than iOS, but I have tested it in apps other than Quine 2. Same result.

**Comment by joshuafontany** [comment]
Hi mpmanti,

Sorry to hear that you have had a problem. I believe, based on another bug report, that some browsers are not applying the "patch" part of the plugin correctly.

As a workaround, I have made a tiddler that is "stand alone", i.e. does not require Evan's ModLoader plugin. I have tested it and attached it to this comment. The big change is that it will replace the core "wiki.js" module of any tiddlywiki.html or node wiki it is loaded into completely.

Download the tiddler, rename it by removeing the ".txt" (so it ends with ".tid") and drag it into your wiki. The save/re-load. Give it a go. Hopefully it gets you back to work.

Thanks for the feedback, and hopefully I can track down the root issue.

Best,
Joshua F

[$__plugins_joshuafontany_jsonmangler_standAlone.tid.txt](https://github.com/joshuafontany/TW5-JsonMangler/files/4534538/__plugins_joshuafontany_jsonmangler_standAlone.tid.txt)

**Comment by joshuafontany** [comment]
I believe I have tracked this down. The wiki.js "patchSource" file was encoded with Windows line-feed (CRLF), and my local Dev TW repo also had Windows encoding, but when compared/patched in Linux line-feed (LF) environment it would fail.

I have normalized all my repos that use this patching method to LR, and regenerated all the affected files (the patches in JsonMangler and Attribute-Modules). A new release, v2.2.1, has been posted.
